.github/workflows/add-hashicorp-contributed-label.yml

```
@@ -17,8 +17,8 @@ on:
 
 jobs:
   add-hashicorp-contributed-label:
-    # Only run if this is NOT coming from a fork of Vault (if this is not true, it's community contributed)
-    if: ${{ github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name }}
+    # Only run if this is NOT coming from a fork of hashicorp/vault (if this is not true, it's community contributed)
+    if: ${{ github.repository == 'hashicorp/vault' && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name) }}
```

```suggestion
    if: github.repository == 'hashicorp/vault' && (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name)
```

The `if` field is special in that everything in it is evaluated as an expression, so we can skip the dollar sign and squiglies 😄 

Here is a link to relevant docs:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif